circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -9,13 +9,16 @@ import cats.effect.Sync
 import cats.implicits._
 import fs2.Chunk
 import io.circe._
-import io.circe.jawn.CirceSupportParser.facade
 import io.circe.jawn._
 import org.http4s.headers.`Content-Type`
 import org.http4s.jawn.JawnInstances
 import org.typelevel.jawn.ParseException
 
 trait CirceInstances extends JawnInstances {
+
+  private val circeSupportParser = new CirceSupportParser(maxValueSize = None, allowDuplicateKeys = false)
```

Comment:
People end up wanting to configure circe so we might want to consider a design that lets them configure these.  But I agree that this is a fine first pass.